### ISSUE TITLE
Implement mechanism for updating reported API version

### ISSUE THREAD

**Issue by hendriksen-mark** [chrivers/bifrost#34]
the api and swversion is outdated and not consistent.
on /clip/v2/resource : 1.60.1960149090
on /api/config : 1.65.0 1965111030

at the time of writing the versions are:
Apiversion: 1.66.0
Swversion: 1966060010

to check version numbers:
https://firmware.meethue.com/v1/checkupdate/?deviceTypeId=BSB002&version=< swversion >
https://www.philips-hue.com/en-gb/support/release-notes/bridge
Between Philips-hue.com and firmware.meethue.com is a delay of around 7 days.
Firmware.meethue.com is in json, for api version only use `1.66` do not include what comes after otherwise the app does not work.

**Comment by chrivers** [comment]
@hendriksen-mark good catch! I've updated the version fields, and made them consistent.

Existing users will need to update their `state.yaml` with the new strings. I'll keep this issue open, until we've figured out a good way to handle this longer-term.

**Comment by hendriksen-mark** [comment]
this is how we do it at diyhue in python.
https://github.com/diyhue/diyHue/blob/master/BridgeEmulator/services/updateManager.py#L11

**Comment by chrivers** [comment]
Yes, that makes sense. But we need some infrastructure for updating this after the `state.yaml` file is first created.

We also need some other user interaction support in the future, like "pressing the link button". I think the diyhue way of having a web ui with a "link" button is pretty promising. It's a good analogy, and it makes sense to the user.

**Comment by chrivers** [comment]
That took a lot longer than I had hoped, but it's almost ready now! :)

I'm reworking some bits now, hopefully I can publish this weekend!

**Comment by chrivers** [comment]
@hendriksen-mark It's ready for testing now: https://github.com/chrivers/bifrost/pull/48 :)
